contentcuration/contentcuration/views/public.py

```diff
@@ -72,7 +71,7 @@ def get_public_channel_lookup(request, version, identifier):
         return HttpResponseNotFound(_("Api endpoint {} is not available").format(version))
     if not channel_list.exists():
         return HttpResponseNotFound(_("No channel matching {} found").format(identifier))
-    return HttpResponse(json.dumps(PublicChannelSerializer(channel_list, many=True).data))
+    return Response(PublicChannelSerializer(channel_list, many=True).data)
```

Comment:
Just want to make sure, does this break any versions of Kolibri since the type returned will be different (string vs. array)?

Reply:
The output of the endpoint should not be any different, `rest_framework.Response` serializes the data to JSON internally. It also returns a correct `Content-Type` header of `application/json`, unlike HttpResponse, which returns `text/html`, which is why I made the change.

Reply:
Will this affect any front-end code? When I did a similar thing changing HttpResponse to rest_framework.Response, I had to remove any JSON.parse instances from the front end to get it to work

Reply:
Is any Studio frontend code using the Kolibri public channel list API? I wasn't able to find any code paths in the Studio frontend using it. 

On a more general note, we should actually look at any places we need to explicitly parse and make fixing those part of our tech debt. It means we're not properly declaring the return type, which will cause problems when writing tests, etc.

Reply:
Alright, I think it should be fine. Worst case, we just revert back to the json dumps code :)